app/code/core/Mage/Api/Model/Resource/Roles.php

```diff
@@ -77,7 +72,7 @@ protected function _beforeSave(Mage_Core_Model_Abstract $role)
     protected function _afterSave(Mage_Core_Model_Abstract $role)
     {
         $this->_updateRoleUsersAcl($role);
-        Mage::app()->getCache()->clean(Zend_Cache::CLEANING_MODE_MATCHING_TAG);
+        Mage::app()->getCache()->clean();
```

Comment:
I sort of wonder why this is here, but since saving roles doesn't happen often I don't think it's an issue.

Reply:
you mean the original line?

Reply:
Yeah, the original line, because it flushes the entire cache, but then why didn't they write `Mage::app()->getCache()->flush()` ?

Reply:
yeah, it's a bit weird, should we change it to flush()?